lib/task/migrate/arUpgradeSqlTask.class.php

```diff
@@ -54,6 +54,7 @@ protected function configure()
       new sfCommandOption('connection', null, sfCommandOption::PARAMETER_REQUIRED, 'The connection name', 'propel'),
       new sfCommandOption('no-confirmation', 'B', sfCommandOption::PARAMETER_NONE, 'Do not ask for confirmation'),
       new sfCommandOption('verbose', 'v', sfCommandOption::PARAMETER_NONE, 'Verbose mode', null),
+      new sfCommandOption('number', 'n', sfCommandOption::PARAMETER_OPTIONAL, 'Run only specific migration number(s)', null),
```

Comment:
I think it would be helpful to indicate in the help text that multiple numbers should be comma-separated.